bibliopixel/main/demo.py

```diff
@@ -143,8 +143,8 @@ def circle(args):
     driver = DriverSimPixel(sum(pixels_per), layout=points)
     led = LEDCircle(driver, rings=rings, maxAngleDiff=0)
 
-    from BiblioPixelAnimations.circle.bloom import CircleBloom
```

Comment:
Dude.... Pull the latest BPA! This exists: https://github.com/ManiacalLabs/BiblioPixelAnimations/blob/master/BiblioPixelAnimations/circle/bloom.py#L5